frontend/src/pages/MapView.tsx

```diff
@@ -72,6 +74,7 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
       swLatitude: sw.lat,
       swLongitude: sw.lng,
       address: address ?? null,
+      municipality: municipality ?? null,
```

Comment:
Is this so the value is `null` but not `undefined`?

Reply:
Yeah. That's so we always send the field along in the query string like `?municipality=`. 

If `undefined` I believe nothing is sent across